package.json

```diff
@@ -39,6 +39,7 @@
     "axios": "0.16.2",
     "circular-json": "0.3.3",
     "lambda-env-vars": "0.4.0",
+    "node-lambda": "~0.17.0",
```

Comment:
Shouldn't this remain a devDependency?

Reply:
Yes. Thanks for pointing that out. Moved it back.